dockers/docker-sonic-mgmt-framework/Dockerfile.j2

```diff
@@ -18,6 +18,7 @@ RUN pip2 install wheel==0.35.1
 RUN pip2 install connexion==1.1.15 \
                 setuptools==21.0.0 \
                 grpcio-tools==1.20.0 \
+                protobuf==4.0.0rc2 \
```

Comment:
This is a yanked version (https://pypi.org/project/protobuf/#history). Should we use a stable version?

Reply:
Changed to use stable version.